.github/workflows/gsbuild.yml

```diff
@@ -0,0 +1,43 @@
+name: GS Test Build
+
+on:
+  pull_request:
+
+jobs:
+  build-roman:
+    runs-on: ubuntu-latest
+    name: Build Roman variable font
+    steps:
+      - name: Check out font project source repository
+        uses: actions/checkout@v4
+        with:
+          repository: googlefonts/googlesans
+          token: ${{ secrets.GS_READ_FONTC }}
+      - name: Install the latest stable Rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+      - name: Install fontc and compile fonts
+        uses: googlefonts/fontc-action@v1
+        with:
+          source-path: source/GoogleSans/GoogleSans.designspace
+          debug: true
```

Comment:
You could just assume we are in the root of our repo and run cargo commands akin to what https://github.com/googlefonts/fontc/blob/main/.github/workflows/rust.yml does?

Reply:
yeah, I guess that was kind of silly, eh?

Reply:
7db4cb1f058d0a01e687312099dd784390dd904f